test/unittests/commands/xmlupload/models/deserialize/test_xmlresource.py

```diff
@@ -0,0 +1,167 @@
+from lxml import etree
+from pytest_unordered import unordered
+
+from dsp_tools.commands.xmlupload.models.deserialise.deserialise_value import IIIFUriInfo
+from dsp_tools.commands.xmlupload.models.deserialise.deserialise_value import XMLBitstream
+from dsp_tools.commands.xmlupload.models.deserialise.deserialise_value import XMLProperty
+from dsp_tools.commands.xmlupload.models.deserialise.deserialise_value import XMLValue
+from dsp_tools.commands.xmlupload.models.deserialise.xmlresource import XMLResource
+from dsp_tools.commands.xmlupload.models.formatted_text_value import FormattedTextValue
+
+
+class Test_XMLResource:
```

Comment:
having one big test class makes having test classes pointless. In pytest (as opposed to unittest), test classes only serve for organization. And having one organizational unit is just as good as having none.  
In this case, I would see logical organizational units to have test classes for the test cases testing `get_restype_...` and `get_properties_(of)_...` (though the naming isn't fully consistent there), and maybe have the rest either top level or in a "sammelbecken" test class